.PLAYWRIGHT_VERSION

```diff
@@ -0,0 +1,4 @@
+# This is the version of playwright compatible with this foxhound Git commit.
+# In case the patches from this version do not apply cleanly, please open an issue!
+# For advice on how to build foxhound with playwright support, see build.sh
+PLAYWRIGHT_VERSION="v1.44.0"
```

Comment:
We could also consider adding the rust version here?
Even if the script is not doing bootstrapping, it might be useful for reference?

Reply:
Hmmm, we try to rely on `./mach bootstrap` here. Once this PR gets merged, I plan to update the Wiki page with an explanation of how to build it with this script. 

We can add the rust version here, but I'm not sure it adds much compared to linking to the [rust update policy](https://firefox-source-docs.mozilla.org/writing-rust-code/update-policy.html) if we don't validate this. Validating this feels quite involved, however.

For example, on my system:
- System wide rust install:
```sh
*(+)[limos:~]> rustc --version
rustc 1.81.0 (eeb90cda1 2024-09-04) (Arch Linux rust 1:1.81.0-1)
```

- rustup default rust:
```sh
*(+)[limos:~]> rustup run 1.74.0-x86_64-unknown-linux-gnu rustc --version
rustc 1.74.0 (79e9716c9 2023-11-13)
```

Mach picks up the latter, but I'm unsure what the heuristic is here.

Reply:
Sounds fine, I remember getting into the situation where my rust version was too recent and I had to downgrade to get the build to work.

But I guess we would just be reproducing what is in the rust update policy link. It would be great if you can update the Wiki with this link!